### DISCUSSION TITLE
How to integrate google ads  (adsense) for web platform.

### DISCUSSION THREAD

**Discussion by cjmling** [necolas/react-native-web#2614] [Q&A]
I successfully integrated google ads for my mobile app using `https://github.com/invertase/react-native-google-mobile-ads`

Now I want to integrate for web.

So in order to display google ads for "web" platform I need to load this right `<script async src="https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>` ?  and then paste ads code on a component which could be `FILE_NAME.web.tsx`

So my question is how I can load script tag for web specifically? Or if there is any other tutorial will be great.

**Comment by mathis-kdio** [comment]
Hi @cjmling, have you found a solution?
